web/src/pages/Settings/Presets/CreatePresetModal/CreatePresetModal.jsx

```diff
@@ -108,7 +110,7 @@ const buildTreeDict = (status, showInactive = false) => {
 
     const statusClone = JSON.parse(JSON.stringify(status));
     const sourceDicts = statusClone.sources
-        .filter((source) => showInactive || getSourceInputType(source) !== "none")
+        .filter((source) => showInactive || getSourceInputType(source) !== "none" && getSourceInputType(source) !== "unknown")
```

Comment:
This closes #980 
Getting to the bottom of why state isn't being set properly on stream close is a different issue (#981)